tests/llmcompressor/transformers/sparsification/test_compress_tensor_utils.py

```diff
@@ -537,7 +542,7 @@ def __init__(self, weights, weight_scale=None, weight_zero_point=None):
 
         # Linear layer without bias
         self.linear = nn.Linear(in_features, out_features, bias=False)
-        self.linear.weight = nn.Parameter(weights, requires_grad=False)
+        self.linear.weight = nn.Parameter(weights, requires_grad=True)
```

Comment:
we need gradients here?

Reply:
When generating a sparsity config, we calculate the global_sparsity, which looks at trainable parameters. The test was skipping this step before so it did not matter previously.